src/MarkDown/PrivacyPolicy.md

```diff
@@ -0,0 +1,21 @@
+# Privacy Policy
+
+Overall our goal is to reduce tracking and data storage on users to promote a safe environment for all users. 
+
+## Tracking and Data Storage
+
+For users who do not have an account:
+
+There is minimal data tracking which primarily includes data collected by CloudFlare such as which pages are accessed, how many requests are made, how many unique users are visiting, and the country of origin of said users. 
+
+For users with an account: 
+
+Data related to your account such as your email and login credentials, as well as content warnings you have submitted are securely stored. These are availible for your viewing discretion in your profile and can be exported there. 
+
+For your safety, your content warnings are NOT stored on MovieMentor's servers, rather saved as a in your browser on your local machine. 
+
+In compliance with regulations such as those presented by the General Data Protection Regulation (GDPR) and California Consumer Protection Act (CCPA) all data is secured and monitored for potential data breaches. Additionally users are provided with the option to delete their account including all data stored, as well as export all of the data stored on an individual.  
```

Comment:
I think a comma after (CCPA) is appropriate

Reply:
so `California Consumer Protection Act (CCPA), all data`

Reply:
same with `Additionally, users are`

Reply:
Correct.